modules/storage/glance.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2022 Red Hat
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	// GlanceExtraVolTypeUndefined can be used to label an extraMount which
+	// is not associated with a specific backend
+	GlanceExtraVolTypeUndefined ExtraVolType = "Undefined"
+	// GlanceExtraVolTypeCeph can be used to label an extraMount which
+	// is associated to a Ceph backend
+	GlanceExtraVolTypeCeph      ExtraVolType = "Ceph"
+
+	// GlanceAPI defines the glance-api group
```

Comment:
done, we can converge to `PropagationType` because we're doing string comparison, no need to have multiple types that are strings and refer to the same pattern